WordPress/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
So far I've tried to manually revert this change each time. It's annoying but I haven't found a way to fix it yet

Reply:
Is this important? Or just annoying because it's different from the default android format for  xml files?

Reply:
Not important afaik